src/taker/api.rs

```diff
@@ -1627,7 +1652,10 @@ impl Taker {
                 .collect::<Vec<MultisigPrivkey>>()
         } else {
             assert!(outgoing_privkeys.is_some());
-            let reply = outgoing_privkeys.as_ref().unwrap().to_vec();
+            let reply = outgoing_privkeys
+                .as_ref()
+                .expect("outgoing privkey expected")
+                .to_vec();
```

Comment:
Its not that simple. The `None` value is also used inside the function with some intricate logic. Can change the logic to handle empty vectors instead of `None`. But not gonna mess with that deep internals here.

Added a TODO for future brave devs.